apps/webapp/src/components/send/layout.tsx

```diff
@@ -13,7 +13,7 @@ export const SendLayout = () => {
     <div className='relative mx-auto grid gap-6 md:grid-cols-2 md:gap-4 xl:max-w-[1276px] xl:grid-cols-3 xl:gap-5'>
       <div className='hidden xl:order-1 xl:block' />
       <Card gradient className='order-2 row-span-2 flex-1 p-5 md:order-1 md:p-4 xl:p-5'>
-        <Tabs tabs={sendTabs} activeTab={pathname} className='mx-auto w-[75%] lg:w-[372px]' />
+        <Tabs tabs={sendTabs} activeTab={pathname} className='mx-auto flex w-[75%] lg:w-[372px]' />
```

Comment:
This was weird. Something about the previous version of the card had made the tabs centered. After my change, they were no longer centered, because they default to being `inline-flex`. This seems like some messy CSS that I eventually want to pull the thread on, but not today :D